doc/contributors-guide/making-and-submitting-changes.md

```diff
@@ -3,9 +3,10 @@
 Everyone is welcome to help improve the content of the Git and Github training resources. There are two repositories linked to the Git and Github project i.e. [learning resource repo](https://github.com/Ifycode/git-github-training-resource) and the [training repo](https://github.com/Ifycode/git-github-training). Interested contributors can follow the guides below when contributing to any of the Git and Github training repositories.
 
 {% hint style="info" %}
- Interested contributors should note that taking the [Git and Github training](https://obiagba-mary.gitbook.io/git-and-github-training) can help you get better at contributing to the Git and Github training project (and any other Github projects they will be taking on in the future).
+ Interested contributors should note that taking the [Git and Github training](https://obiagba-mary.gitbook.io/git-and-github-training) can help you get better at contributing to the Git and Github training project (and any other Github projects they will be taking on in the future). As contributors you can indicate, share as you are contributors to the [Git and GitHub gitbook training resource](https://obiagba-mary.gitbook.io/git-and-github-training) on your portfolio, as long as you are among contributors listed in the [contributors section](https://github.com/Ifycode/git-github-training-resource#contributors-). Anyone who has contributed and has not been added to the contributors list should [notify repo admins through the learning resource repo's discussion.](https://github.com/Ifycode/git-github-training-resource/discussions/2)
```

Comment:
@oyenuga17 The task says to add it as hint, not merge it into an existing hint.

We want all hints to separate because the info they convey are not the same.

Reply:
i have made correction please help me review @Ifycode